packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -177,6 +183,8 @@ import FormField from '@justeat/f-form-field';
 import '@justeat/f-form-field/dist/f-form-field.css';
 import ErrorMessage from '@justeat/f-error-message';
 import '@justeat/f-error-message/dist/f-error-message.css';
+import VLink from '@justeat/f-link';
```

Comment:
Any reason why this is VLink and not FLink?

Reply:
It seems to be the new standard from the components generator. We have `VPopOver` as well